CommonTools/RecoAlgos/plugins/ME0MuonTrackCollProducer.cc

```diff
@@ -0,0 +1,73 @@
+
+#include "CommonTools/RecoAlgos/plugins/ME0MuonTrackCollProducer.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/MuonDetId/interface/CSCDetId.h"
+#include "DataFormats/MuonDetId/interface/DTChamberId.h"
+#include "DataFormats/MuonDetId/interface/MuonSubdetId.h"
+//#include "DataFormats/MuonReco/interface/MuonSelectors.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h" 
+#include "RecoMuon/MuonIdentification/plugins/ME0MuonSelector.cc"
+#include <FWCore/Framework/interface/ESHandle.h>
+
+
+// #include "Geometry/GEMGeometry/interface/ME0Geometry.h"
+// #include <Geometry/GEMGeometry/interface/ME0EtaPartition.h>
+// #include <Geometry/Records/interface/MuonGeometryRecord.h>
+// #include <DataFormats/MuonDetId/interface/ME0DetId.h>
+
+#include <sstream>
+
```

Comment:
move the contents of ME0MuonTrackCollProducer.h here in the .cc file and the contents of the SealModule.cc file as well.
These extra two files are not really needed.